jubako/src/main/java/com/justeat/jubako/ContentDescriptionCollection.kt

```diff
@@ -6,7 +6,7 @@ package com.justeat.jubako
  */
 class ContentDescriptionCollection(var listener: Listener? = null) {
 
-    private val mContentDescriptions = mutableListOf<ContentDescription<*>>()
+    internal var mContentDescriptions = mutableListOf<ContentDescription<*>>()
```

Comment:
Should this be `contentDescriptions` without the Hungarian notation?

Reply:
@Electryc yeh, overspill from legacy java! thanks